apps/studio/components/interfaces/SQLEditor/MonacoEditor.tsx

```diff
@@ -138,6 +143,36 @@ const MonacoEditor = ({
         // - https://github.com/microsoft/monaco-editor/issues/2229
         // - https://github.com/microsoft/monaco-editor/issues/2503
         // fixedOverflowWidgets: true,
+        suggest: {
```

Comment:
this is all a bit crazy.... I don't see a single option to toggle them all though.

Reply:
ha